Releases/chrome/runSwitch.js

```diff
@@ -2,7 +2,7 @@ function runSwitchjs() {
   alertify
   .okBtn("Accept")
   .cancelBtn("Deny")
-  .confirm("After clicking on 'Ok', Starring will start, leave the tab open, we'll let you know after the starring process. Feel free to jump on another tabs :)", function (ev) {
+  .confirm("After clicking on 'Accept', Starring will start, leave the tab open, we'll let you know after the starring process. Feel free to jump on another tabs :)", function (ev) {
```

Comment:
"Ok" is fine.

Reply:
Yes, then let it be 'Ok'. I thought the 'Ok' referred to the old style of using alert box. The new one said 'Accept'.